src/main/java/cz/muni/fi/pv286/mmt/model/Options.java

```diff
@@ -0,0 +1,163 @@
+package cz.muni.fi.pv286.mmt.model;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Optional;
+
+/**
+ * This class encapsulates all possible options.
+ */
+public class Options {
+    private IoFormat inputFormat;
+    private Optional<IoOption> inputFromToOption = Optional.empty();
+    private IoFormat outputFormat;
+    private Optional<IoOption> outputFromToOption = Optional.empty();
+    private InputStream inputFile = System.in;
+    private OutputStream outputFile = System.out;
+    private String delimiter = "\n";
+    private Optional<BracketType> bracketType = Optional.empty();
+    private boolean wasFromOptionSet = false;
+    private boolean wasToOptionSet = false;
+    private boolean wasBracketSet = false;
+    private boolean wasDelimiterSet = false;
+
+    /**
+     * Gets the input format.
+     */
+    public IoFormat getInputFormat() {
+        return inputFormat;
+    }
+
+    /**
+     * Sets the input format.
+     */
+    public void setInputFormat(IoFormat inputFormat) {
+        if (inputFormat == IoFormat.INT && !wasFromOptionSet) {
+            inputFromToOption = Optional.of(IoOption.BIG);
+        }
+        if (inputFormat == IoFormat.BITS && !wasFromOptionSet) {
+            inputFromToOption = Optional.of(IoOption.LEFT);
+        }
+        this.inputFormat = inputFormat;
+    }
+
+    /**
+     * Get the input FromToOption.
+     */
+    public Optional<IoOption> getInputFromToOption() {
+        return inputFromToOption;
+    }
+
+    /**
+     * Sets the input FromToOption.
+     */
+    public void setInputFromToOption(FromToOption inputFromToOption) {
+        this.inputFromToOption = Optional.of(IoOption.from(inputFromToOption));
+        wasFromOptionSet = true;
+    }
+
+
+    /**
+     * Sets the bracket type option.
+     */
+    public void setBracketOption(BracketType bracketType) {
+        this.bracketType = Optional.of(bracketType);
+        wasBracketSet = true;
+    }
```

Comment:
**issue:** Naming of setter

Setters and getters should be named with parameters name (setBracketType).